src/run_tribler.py

```diff
@@ -83,6 +83,8 @@ def init_boot_logger():
     logger.info(f'Root state dir: {root_state_dir}')
 
     api_port = os.environ.get('CORE_API_PORT')
+    api_port = int(api_port) if api_port else None
```

Comment:
The previous code version did not handle the situation when the `CORE_API_PORT` value is not set for the Core process.